src/styles/bulk_concepts.scss

```diff
@@ -215,12 +215,18 @@
   align-items: center;
 }
 
-.search-ul{
```

Comment:
What is the reason behind renaming it?

Reply:
Previously the <div> tag had the same className as the <ul> tag on which I had to make the changes in CSS.
In order to apply the changes specifically to the <ul> tag, I had to change its className.